src/pages/LoginOld.js

```diff
@@ -98,13 +98,13 @@ const Login = () => {
         </Text>
       </Box>
 
-      <Box mb={3}>
-        <Text as="h2" mb={2}>
+      <Box mb={3} maxWidth="90%">
```

Comment:
Can you check the box width on mobile view?
I believe setting up `maxWidth` with fixed `px` might be better choice, but play around it!

Reply:
I did try it out with `px` but on one screen it would go too far to the right, and the other screen become too cut off. I'll try out a few variations...

Reply:
Made it 1600px seems to work well...
![image](https://github.com/bcgov/platform-services-registry-web/assets/145396323/473897a1-d015-4a0c-961e-a682a23649cf)